src/runtime/qurt_thread_pool.cpp

```diff
@@ -117,60 +115,23 @@ WEAK int halide_do_par_for(void *user_context,
         // initializing this mutex.
         qurt_mutex_init(mutex);
     }
-
-    wrapped_closure c = {closure, qurt_hvx_get_mode()};
-
-    // Set the desired number of threads based on the current HVX
-    // mode.
-    int old_num_threads =
-        halide_set_num_threads((c.hvx_mode == QURT_HVX_MODE_128B) ? 2 : 4);
+    int old_num_threads = halide_set_num_threads(HEXAGON_DEFAULT_NUM_THREADS);
```

Comment:
I think we should just not do this. We already have this: https://github.com/halide/Halide/blob/fca6a541c99a14f9159c4f290b0bd704f0372e44/src/runtime/qurt_thread_pool.cpp#L16 so it should just do the right thing if we do nothing.